### ISSUE TITLE
automated vs semi-automated vs manual testing methodology

### ISSUE THREAD

**Issue by giacomo-petri** [act-rules/act-rules.github.io#1882]
Considering the purpose of the ACT Rules

> The purpose of the Accessibility Conformance Testing (ACT) effort is to establish and document rules for testing the conformance of web content to accessibility standards, such as Web Content Accessibility Guidelines (WCAG). These test rules address automated, semi-automated, and manual testing. ACT makes accessibility testing more transparent, and thus reduces confusion caused by different interpretations of accessibility guidelines.

might be interesting (and helpful) to clearly define which rules/tests can be implemented/tested in a fully automated way and which ones instead require a manual evaluation (or a combination of both in case of a composite rule).

For example, the following rules can be evaluated automatically by an accessibility test tool:
- [HTML page lang attribute has valid language tag](https://www.w3.org/WAI/standards-guidelines/act/rules/bf051a/) 
- [HTML page lang and xml:lang attributes have matching values](https://www.w3.org/WAI/standards-guidelines/act/rules/5b7ae0/)

These require a manual verification:
- [HTML graphics contain no text](https://www.w3.org/WAI/standards-guidelines/act/rules/0va7u6/proposed/)
- [Video element content is media alternative for text](https://www.w3.org/WAI/standards-guidelines/act/rules/ab4d13/proposed/)

With a high number of rules, results might be also interesting to demonstrate and show (in numbers) which is the difference between a manual and automatic test or how much a fully automated test can contribute in an accessibility review.

**Comment by WilcoFiers** [comment]
Thank you for the suggestion. Originally ACT Rules were explicitly designed for one of the three test approaches. We moved away from that idea several years ago because we found that as technologies advance, these things evolve.

Current thinking on this question is that, rather than define what test mode a rule is in, we can simply track what implementors have done, and have that be the measure of if the rule can be autoamted, semi-automated, or manual. This was very recently added to the rules table at the bottom of each rule. This isn't live yet, but it should be soon. You can see a preview of it here:

https://wai-wcag-act-rules.netlify.app/standards-guidelines/act/rules/73f2c2/#implementations

It might also be interesting to have an option to show this in the rules list itself too. The rules list page definitely needs to be updated. Just having a few lists there isn't idea. But I believe what you're asking for is already in the preview, and will be on the WAI website the next time it updates. I think we can consider this issue addressed.
